src/client/decorators/MetaDecorator/DiagramDesigner/MetaDecorator.DiagramDesignerWidget.js

```diff
@@ -610,7 +610,8 @@ define([
                 len: LEN
             };
 
-            if (connType && connType === MetaRelations.META_RELATIONS.INHERITANCE) {
+            if (connType && (connType === MetaRelations.META_RELATIONS.INHERITANCE ||
```

Comment:
cond2 || cond3 doesn't fit in a single line.
is it ok if I break into 3 lines?
if(cond1 &&
   (cond2 ||
   cond3)) {